website/documentation/tutorials/oidc-login/_index.md

```diff
@@ -0,0 +1,209 @@
+
+1. In your tenant, set up a native client application that will use the authentication:
+
+   ![application](./application.png)
+
+2. Configure the client to have a callback url of `http://localhost:8000`. This callback connects to your local `kubectl oidc-login` plugin:
+
+   ![application](./callback.png)
+
+3. Note down the following parameters:
+
+    * Domain or Issuer url. It [must](https://openid.net/specs/openid-connect-core-1_0.html#Terminology) be an `https`-secured endpoint (In case of Auth0, notice the trailing `/` at the end).
+    * Client ID
+    * Client Secret
+
+4. Verify that `https://<Issuer>/.well-known/openid-configuration` is reachable.
+
+5. Create some users (or connect to a user store):
+
+   ![application](./user.png)
+
+   Notice that the users must have a *verified* email address. In doubt, just override that setting manually.
+
+## Configure a local kubectl oidc-login
+
+1. Install the `kubectl` plugin [oidc-login](https://github.com/int128/kubelogin). We highly recommend the [krew](https://github.com/kubernetes-sigs/krew) install tool, which also makes other plugins easily available.
+
+    ```
+    $ kubectl krew install oidc-login
+    Updated the local copy of plugin index.
+    Installing plugin: oidc-login
+    CAVEATS:
+    \
+    |  You need to setup the OIDC provider, Kubernetes API server, role binding and kubeconfig.
+    |  See https://github.com/int128/kubelogin for more.
+    /
+    Installed plugin: oidc-login
+    ```
+
+2. Prepare a `kubeconfig` for later use:
+   
+    ```
+    $ cp ~/.kube/config ~/.kube/config-oidc
+    ``` 
+
+3. Modify the configuration as follows:
+   
+    ```yaml
+    apiVersion: v1
+    kind: Config
+
+    ...
+
+    contexts:
+    - context:
+        cluster: shoot--project--mycluster
+        user: my-oidc
+      name: shoot--project--mycluster
+
+    ...
+
+    users:
+    - name: my-oidc
+      user:
+        auth-provider:
+          config:
+            client-id: <Client ID>
+            client-secret: <Client Secret>
+            idp-issuer-url: "https://<Issuer>/"
+            extra-scopes: email,offline_access,profile
+          name: oidc
+    ``` 
+
+4. Ensure that the modified context is the active context `current-context: shoot--project--mycluster`.
+
+## Configure the shoot cluster
+
+Modify the shoot cluster YAML as follows:
+
+```yaml
+kind: Shoot
+apiVersion: garden.sapcloud.io/v1beta1
+metadata:
+  name: mycluster
+  namespace: garden-project
+...
+spec:
+  kubernetes:
+    kubeAPIServer:
+      oidcConfig:
+        clientID: <Client ID>
+        issuerURL: "https://<Issuer>/"
+        usernameClaim: email
+```
+
+This change of the `Shoot` manifest triggers a reconciliation. Once the reconciliation is finished, your OIDC configuration is applied. It does *not* invalidate other certificate based authentication methods. Wait for Gardener to reconcile the change. It can take up to 5 minutes.
```

Comment:
```suggestion
This change of the `Shoot` manifest triggers a reconciliation. Once the reconciliation is finished, your OIDC configuration is applied. It does *not* invalidate other certificate-based authentication methods. Wait for Gardener to reconcile the change. It can take up to 5 minutes.
```